config/locales/en.yml

```diff
@@ -108,7 +108,8 @@ en:
       <p class="govuk-body">We do this for your security. We've deleted all the details you entered to protect your data.</p>
   leave_this_website:
     link_text: "Leave this site"
-    link_href: "https://www.bbc.co.uk/"
+    link_href: "clear-session?ext_r=true"
```

Comment:
Does this need to be "/clear-session?ext_r=true"? Also, I wonder if using `clear_session_path(ext_r: true)` works here?

Reply:
Yeah, I think you're right that it should have a leading `/`.

I don't think the `clear_session_path` helper will work here because we're not in a ruby context, but we could do that in the view instead of reading the href from the localisation file.